horizons/world/units/collectors/collector.py

```diff
@@ -406,8 +406,8 @@ def transfer_res_to_home(self, res, amount):
 		remnant = self.get_home_inventory().alter(res, amount)
 		#assert remnant == 0, "Home building could not take all resources from collector."
 		remnant = self.get_component(StorageComponent).inventory.alter(res, -amount)
-		assert remnant == 0, "%s couldn't give all of res %s; remnant: %s; inventory: %s" % \
-		       (self, res, remnant, self.get_component(StorageComponent).inventory)
+		assert remnant == 0, "{} couldn't give all of res {}; remnant: {}; inventory: {inv}".format(
```

Comment:
Why did you name make the last argument (`inv`) to format? Just use `{}` like the others.